tracing-agent/src/main/java/io/strimzi/tracing/agent/OpenTelemetryTracing.java

```diff
@@ -20,7 +20,8 @@ public void initialize() {
         String serviceName = System.getenv("OTEL_SERVICE_NAME");
         if (serviceName != null) {
             LOGGER.info("Initializing OpenTelemetry tracing with service name {}", serviceName);
-            System.setProperty("otel.metrics.exporter", "none"); // disable metrics
+            System.setProperty("otel.metrics.exporter", "none"); // disable otel metrics exporter. This has no effect on metrics in the bridge.
+            System.setProperty("otel.logs.exporter", "none"); // disable otel logs exporter. This has no effect on logging in the bridge.
```

Comment:
@ppatierno I'm not sure what context the tracing agent is used, but its certainly not `in the bridge` 😜 How about
```suggestion
            System.setProperty("otel.metrics.exporter", "none"); // disable otel metrics exporter. This has no effect on metrics in Strimzi.
            System.setProperty("otel.logs.exporter", "none"); // disable otel logs exporter. This has no effect on logging in Strimzi.
```

Reply:
Yeah sure. Blind copy/paste grrrrrrrr
Please update this way.